### PR TITLE
Minor documentation and GUI fixes

### PR DESCRIPTION
**Pull request by lassoan**

### CODE REVIEW

**Review by jcfr — APPROVED** [review]

File: Docs/user_guide/getting_started.md [hunk 1]
```diff
@@ -163,7 +163,7 @@ You can customize views (show orientation marker, ruler, change orientation, tra
 
 #### Save data
 
-Data sets loaded into the application can be saved using Save data dialog or exported to DICOM format using DICOM module. Detailes are described in [Data loading and saving section](data_loading_and_saving.md).
+All data in the scene can be saved at once using `File` menu -> `Save data`, or selected data sets can be exported from `Data` module by right-clicking and selecting `Export to file...` or `Export to DICOM...` . Details are described in [Data loading and saving section](data_loading_and_saving.md).
```
Comment (jcfr) [inline Docs/user_guide/getting_started.md:166]:
```suggestion
All data in the scene can be saved at once using `File` menu -> `Save data`, or selected data sets can be exported from the `Data` module by right-clicking and selecting `Export to file...` or `Export to DICOM...` . Details are described in the [Data loading and saving section](data_loading_and_saving.md).
```

File: Docs/user_guide/data_loading_and_saving.md [hunk 2]
```diff
@@ -20,13 +20,42 @@ More details are provided in the [DICOM module documentation](modules/dicom.md).
...
 
+### Export data
+
+To export selected data sets - for sharing with others or for loading into other applications: go to `Data` module, right-click on an item, and choose `Export to file...`. Settings used for exporting (file format, filename, options) do not modify settings used for saving.
+
+![](https://github.com/Slicer/Slicer/releases/download/docs-resources/data_loading_and_saving_export_dialog.png)
+
+:::{tip}
+
+Multiple nodes can be exported at once by placing them into a folder and then exporting the folder. When exporting an entire folder hierarchy then `Export folder structure` option can be enabled to recreate the subject hierarchy folder structure as directory structure in the output directory.
```
Comment (jcfr) [inline Docs/user_guide/data_loading_and_saving.md:55]:
```suggestion
Multiple nodes can be exported at once by placing them into a folder and then by exporting the folder. When exporting an entire folder hierarchy the `Export folder structure` option can be enabled to have the directory structure in the output directory match the subject hierarchy folder structure.
```

File: Docs/user_guide/data_loading_and_saving.md [hunk 3]
```diff
@@ -20,13 +20,42 @@ More details are provided in the [DICOM module documentation](modules/dicom.md).
...
+
+### Save data
+
+Save the entire scene - preserves all data and settings:
+- in application menu `File` -> `Save Data`, or
+- `Save Data` toolbar button ![](../../Base/QTGUI/Resources/Icons/Medium/SlicerSave.png)
+
+:::{tip}
+
+The entire workspace, including all data and settings can be saved into a single, independent, self-contained .mrb file by clicking the small package icon at the top-left corner. A new copy of all files is written and zipped into a single file, therefore saving takes longer time than an incremental saving of only modified files.
```
Comment (jcfr) [inline Docs/user_guide/data_loading_and_saving.md:41]:
```suggestion
The entire workspace, including all data and settings can be saved into a single, independent, self-contained `.mrb` file by clicking on the small package icon at the top-left corner. A new copy of all files is written and zipped into a single file, therefore saving takes longer time than an incremental saving of only the modified files.
```

File: Docs/user_guide/data_loading_and_saving.md [hunk 4]
```diff
@@ -20,13 +20,42 @@ More details are provided in the [DICOM module documentation](modules/dicom.md).
...
+
+To load data:
+- drag&drop file on the application window, or
+- in application menu: `File` -> `Add Data` (or `Add Data` toolbar button) ![](../../Base/QTGUI/Resources/Icons/Medium/SlicerLoadData.png)
+
+![](https://github.com/Slicer/Slicer/releases/download/docs-resources/data_loading_and_saving_load_dialog.png)
+
+### Save data
+
+Save the entire scene - preserves all data and settings:
```
Comment (jcfr) [inline Docs/user_guide/data_loading_and_saving.md:35]:
```suggestion
To save the entire scene - and preserves all data and settings:
```

**Comment by lassoan** [comment]
Thanks for the review comments @jcfr, I've accepted all of them.

**Pull request merged**